pkg/operator/configobserver/featuregates/observe_featuregates.go

```diff
@@ -124,20 +109,5 @@ func (f *featureFlags) getWhitelistedFeatureNames(fg *configv1.FeatureGate) ([]s
 		newConfigValue = append(newConfigValue, formatDisabledFunc(disable))
 	}
 
-	return newConfigValue, nil
-}
-
-func FeaturesGatesFromFeatureSets(fg *configv1.FeatureGate) ([]string, []string, error) {
```

Comment:
note to self, you still want this for operators that have items on the bootstrap host.

Reply:
> note to self, you still want this for operators that have items on the bootstrap host.

no you won't.  They'll update to read the manifest from the installer.